Jenkinsfile

```diff
@@ -82,28 +82,33 @@ void setTestsresults() {
     }
 }
 
-void runTest(String TEST_NAME, String CLUSTER_PREFIX) {
+void runTest(String TEST_NAME, String CLUSTER_PREFIX, String MYSQL_VERSION) {
     def retryCount = 0
+    def testNameWithMysqlVersion = sh(script: "echo ${TEST_NAME}-${MYSQL_VERSION} | tr '.' '-'", , returnStdout: true).trim()
```

Comment:
Do we need to call shell script for assigning this, we have both parameters in groovy function call, can't we just do it with groovy?
(I'm not saying that we should, but if it's not too complicated why not)